### PR TITLE
Test for all contrib methods

### PR DESCRIPTION
**Pull request by RomainGoussault**

### CODE REVIEW

File: tests/contrib_end_to_end_test.py [hunk 1]
```diff
@@ -55,3 +56,35 @@ def test_mnist_contrib(self):
...
             assert small_dataset_score.values < big_dataset_score.values
+
+    def test_all_contrib_methods(self):
+        """
+        Test all available contributivity methods on mnist
+        """
+
+        all_methods = constants.CONTRIBUTIVITY_METHODS.copy()
+        all_methods.remove('AIS_Kriging_S')  # This one fails
+
```
Comment (arthurPignet) [inline tests/contrib_end_to_end_test.py:67]: Why not removing IS reg S ?

**Review by arthurPignet — COMMENTED** [review]
Really nice ! How does it take to run ?

File: tests/contrib_end_to_end_test.py (hunk 1 above)
Comment (RomainGoussault) [reply tests/contrib_end_to_end_test.py:67]: Why we should remove them ?

**Comment by RomainGoussault** [comment]
> Really nice ! How long does it take to run ?

It's 10 min now for all the end to end tests

File: tests/contrib_end_to_end_test.py (hunk 1 above)
Comment (arthurPignet) [reply tests/contrib_end_to_end_test.py:67]: You specified another test with it, and commented that this method would not start with less than 4 partners

**Review by arthurPignet — APPROVED** [review]

**Pull request merged**